### ISSUE TITLE
Does the parser support UnitOfMeasure?

### ISSUE THREAD

**Issue by t-sch** [geostyler/geostyler-sld-parser#892]
Hello,
I wonder whether the parser supports uom in the Symbolizer element,
I got a SLD 1.1 style containing this LineSymbolizer

```
          <se:LineSymbolizer uom="http://www.opengeospatial.org/se/units/metre">
            <se:Stroke>
              <se:SvgParameter name="stroke">#333333</se:SvgParameter>
              <se:SvgParameter name="stroke-width">1600</se:SvgParameter>
              <se:SvgParameter name="stroke-linejoin">round</se:SvgParameter>
              <se:SvgParameter name="stroke-linecap">round</se:SvgParameter>
            </se:Stroke>
          </se:LineSymbolizer>
```

and I get verrrry wide lines so that the screen is filled. Another LineSymbolizer in the same style without uom and a small width works fine.
I read the SLD style and wrote it to a MB Style and displayed it in MapLibre.

**Comment by KaiVolland** [comment]
Yes, you can pass it to the constructor of the Parser: https://github.com/geostyler/geostyler-sld-parser/blob/master/src/SldStyleParser.ts#L61-L68

It requires `SLDVersion` to be set to `1.1.0`.

But i'm actually not sure if the mapbox-styler-parser already supports this.

**Comment by t-sch** [comment]
Thanks. I tried setting the version to 1.1.0. The symbolizerUnits setting is a global setting in the ctor, right?
My style has a per symbolizer setting (some use it and some not). That works in geostyler preview.
The SLDparser readStyle output has a 1600 width and no hint of a changed unit. should there be a unit member somewhere?

**Comment by KaiVolland** [comment]
Ok. We will leave this issue open then. This feature is more or less in an experimental stage.
Would be helpful if you could open a PR with a failing SLD example in https://github.com/geostyler/geostyler-sld-parser/tree/master/data/slds/1.1 .
